system/ee/legacy/libraries/Xmlrpc.php

```diff
@@ -620,10 +620,16 @@ public function parseResponse($fp)
         $this->xh[$parser_name]['valuestack'] = array();
         $this->xh[$parser_name]['isf_reason'] = 0;
 
-        xml_set_object($parser, $this);
         xml_parser_set_option($parser, XML_OPTION_CASE_FOLDING, true);
-        xml_set_element_handler($parser, 'open_tag', 'closing_tag');
-        xml_set_character_data_handler($parser, 'character_data');
+        xml_set_character_data_handler($parser, function($parser, $cdata) {
+            return $this->character_data($parser, $cdata);
+        });
+        xml_set_element_handler($parser, function($parser, $tag, $attributes) {
```

Comment:
This will require PHP 8.0 to work; I dont think we can use closures here.

Reply:
This was a lint error. If we're confident this is fine in PHP 7, I'll mark as resolved.